config/deploy.rb

```diff
@@ -17,31 +16,30 @@
   color: :auto,
   truncate: :auto
 
-# Default value for :pty is false
-# set :pty, true
-
 set :linked_files, fetch(:linked_files, [])
   .push(
     'server/.env'
   )
 
 set :linked_dirs, fetch(:linked_dirs, [])
   .push(
-    'log',
+    'server/storage/logs',
     'client/app/locales/custom',
     'client/app/sass/custom'
   )
 
 # Default value for default_env is {}
-# set :default_env, { path: "/opt/ruby/bin:$PATH" }
+set :default_env, {
+  path: "/usr/local/rvm/gems/ruby-2.3.0/bin:/usr/local/rvm/gems/ruby-2.3.0@global/bin:/usr/local/rvm/rubies/ruby-2.3.0/bin:$PATH"
```

Comment:
It is a bit fragile, but was necessary for the chef/capistrano stuff to work together.